src/views/Layout.jsx

```diff
@@ -1,28 +1,39 @@
 import { Outlet, NavLink } from 'react-router-dom';
 
-import './Layout.css';
-
 export function Layout() {
 	return (
 		<>
 			<div className="Layout">
 				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
+					<h1>PREDICT-A-BUY</h1>
+					<nav className="Nav">
+						<NavLink to="/" className="Nav-link">
+							Home
+						</NavLink>
+						<NavLink to="/list" className="Nav-link special">
+							Your List
+						</NavLink>
+						<NavLink to="/add-item" className="Nav-link">
+							Add Item
+						</NavLink>
+					</nav>
 				</header>
-				<main className="Layout-main">
-					<Outlet />
-				</main>
-				<nav className="Nav">
-					<NavLink to="/" className="Nav-link">
-						Home
-					</NavLink>
-					<NavLink to="/list" className="Nav-link">
-						List
-					</NavLink>
-					<NavLink to="/add-item" className="Nav-link">
-						Add Item
-					</NavLink>
-				</nav>
+				<div className="wrapper">
+					<main className="Layout-main">
+						<Outlet />
+					</main>
+					<footer>
+						Created by <a href="https://github.com/hansontram">Hanson Tram</a>,{' '}
+						<a href="https://github.com/lizzypine">Lizzie Pine</a>,{' '}
```

Comment:
I'm in the non-traditional Lizzy-with-a-"y" camp. Mind updating the spelling?

Reply:
omg of course! I'm sorry!